strings/src/main/res/values-de/strings.xml

```diff
@@ -665,7 +664,7 @@
   <string name="analytics_summary">Anonyme Nutzungsdaten helfen dem ODK-Team Fehlerbehebungen und neue Funktionen zu priorisieren.</string>
   <string name="form_metadata_title">Formular-Metadaten</string>
   <string name="form_metadata">Formular-Metadaten</string>
-  <string name="form_metadata_summary">Diese Werte werden Formularen hinzugefügt, die Usernamen, E-Mail und/oder durch das Telefon vorbelegte Felder beinhalten, um die Person zu identifizieren, die die Daten eingibt.</string>
```

Comment:
Did you make this change in Transifex? It'll just come back next time we update translations otherwise!